feat: adds Navigation context en (#848)
ZupIT/beagle-docs: content/en/api/actions/navigate/popstack.md

---
title: PopStack
weight: 60
description: Describes the PopStack action and its atributes
---

---

## What is it?

> The ``PopStack`` is an navigation action that removes the stack of screens where the currently screen is displayed on.

Reviewer: ```suggestion
The ``PopStack`` is a navigation action that removes the stack of screens where the current screen is displayed on.
```

The ``PopStack`` structure is:

| **Attribute** | **Type** | Required | **Definition** |
| :----------- | :-------------------------------------------- | :---------: | :----------------- |
| navigationContext | ​[NavigationContext]({{< ref path="/api/actions/navigate/navigationcontext" lang="pt" >}})​ | | Navigation Context sent from the previous screen. |

## How to use it?

> In this example, we have a screen with a button that, when clicked, "kills" the stack of screens where the current screen is on.

Reviewer: ```suggestion
In this example, there is a screen with a button that, when clicked, "kills" the stack of screens where the current screen is on.
```

> To test this you need to navigate to this screen example, in a stack already built, and click on the button. This will destroy the entire stack

Reviewer: ```suggestion
To test it you need to navigate to this screen example, in a stack already built, and click on the button. This will destroy the entire stack
```
